### ISSUE TITLE
CHANGELOG update process

### ISSUE THREAD

**Issue by Mouvedia** [LightTable/LightTable#2247]
This enhancement ticket covers the streamlining to the steps 2 and 3 of the [release checklist](https://github.com/LightTable/LightTable/wiki/Release-Checklist). This is not urgent but could help accelerate the overall process. In that regard it is related to #2133

It could be labeled nice-to-have-but-can-live-without.

## Add changelog to CHANGELOG.md

As explained on http://keepachangelog.com/ this could be facilitated using an [Unreleased] section. That would require updating the guidelines of [for-committers.md](https://github.com/LightTable/LightTable/blob/master/doc/for-committers.md).
Once a new version is ready we could use https://github.com/mtdowling/chag if we are really lazy.

## Copy changelog to GH release draft

This can be automatically done using https://github.com/mattbrictson/chandler

**Comment by kenny-evitt** [comment]
For anyone curious, we currently only update the changelog.md document when we're ready to create a new release:

 - [History for deploy/core/changelog.md - LightTable/LightTable](https://github.com/LightTable/LightTable/commits/master/deploy/core/changelog.md)

I'm not averse to updating it with each PR (or commit), where the relevant change is significant and visible for users. @rundis @sbauer322 – thoughts?

[*chag*](https://github.com/mtdowling/chag) does look pretty cool. I'm open to adopting its recommended workflow in conjunction with updating the CHANGELOG document with every PR/commit.

[*chandler*](https://github.com/mattbrictson/chandler) tho is, I think, overkill for us. We have to (or at least are) manually create a new release in GitHub so copy+paste-ing the relevant portion of the CHANGELOG to the release notes in GitHub isn't really much of a burden.

**Comment by Mouvedia** [comment]
changelog tools: olivierlacan/keep-a-changelog#67

**Comment by sbauer322** [comment]
I'm late to this party as I somehow missed my highlight - despite reading this issue several times - but adding and maintaining an unreleased section in the changelog seems like a good idea. It would ease pressure during release times and seems like a positive change.

chandler does feel like overkill and chag is cool but I'm not sure either provide enough benefits to warrant adding to our workflow. 

What I would find beneficial is something that could update our changelog with each merge. For instance, if a Light Table maintainer added a comment at the end of a PR to the effect of "[CHANGELOG - UNRELEASED]: Fixed bug foo bar - issue #0000" and that was automatically added to the changelog... that would be easy and useful!

@rundis any thoughts?
